include/openssl/pemerr.h

```diff
@@ -32,17 +30,19 @@ int ERR_load_PEM_strings(void);
 # define PEM_F_D2I_PKCS8PRIVATEKEY_FP                     121
 # define PEM_F_DO_B2I                                     132
 # define PEM_F_DO_B2I_BIO                                 133
-# define PEM_F_DO_BLOB_HEADER                             134
+# define PEM_F_DO_BLOB_HEADER                             147
```

Comment:
Interesting this seems to be 134 since 2017 at least.
But the script changes it to 147.

Reply:
If I revert your changes to the auto generated files, run make update, things work for me.

Reply:
Did you maybe cherry-pick the auto generated files from master?

Reply:
magic, It may have started because I cherry-picked, but the define
was missing in the header file, so X509err did not compile